Patches/Kit's Gunpowder Weapons/Defs/Ammo/AmmoCategoryDefs/AmmoCategories_Medieval.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
+	<CombatExtended.AmmoCategoryDef>

```

Comment:
There already are musket balls in the Combat Extended, no need to create a duplicate.

Reply:
My bad, replaced it with CE's.